Changes default Red Hat (rhel) logo to current logo "The hat"

helmchen opened the pull request:
The default logo for RHEL has been changed to the new logo "the hat". The previous logo is sill available as alternate logo `shadowman` 


## Description
- Implementation of the the new logo
- renaming the previous logo to LOGO_SHADOWMAN
- Providing alias for the current logo as LOGO_HAT


## Reason and / or context
Cosmetic reasoons :-)


## How has this been tested ?
Visually tested using following commands:

python3 -m archey -d rhel 
python3 -m archey -d rhel -l shadow man

on a Mac (macOS 13.6.3 x86_64; Darwin 22.6.0) from macOS terminal as well as from inside VSCode.


## Types of changes :
<!--- What types of changes does your code introduce ? Put an `X` in all the boxes that apply : -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Typo / style fix (non-breaking change which improves readability)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist :
<!--- Put an `X` in all the boxes that apply : -->
- [ ] \[IF NEEDED\] I have updated the _README.md_ file accordingly ;
- [ ] \[IF NEEDED\] I have updated the test cases (which pass) accordingly ;
- [ ] \[IF BREAKING\] This pull request targets next Archey version branch ;
- [X] My changes looks good ;
- [X] I agree that my code may be modified in the future ;
- [X] My code follows the code style of this project ([PEP8](https://www.python.org/dev/peps/pep-0008/)).

HorlogeSkynet commented:
Hey @helmchen, awewome PR ! Many thanks.

I've debased your branch and fixed a remaining tabulation.
Moreover, "empty logo lines" are _disallowed_ (but this test case is broken for alternative logos) so I've removed the first one you initially added.

Tell me whether you're OK with the proposed changes 🙂

HorlogeSkynet commented:
(test case has been fixed on master [see 8817077a5ef61a3ed327a94a78ea637f7c707933], I've rebased again your branch :pray:)

helmchen commented:
> Hey @helmchen, awewome PR ! Many thanks.
> 
> I've rebased your branch and fixed a remaining tabulation. Moreover, "empty logo lines" are _disallowed_ (but the corresponding test case is broken for alternative logos) so I've removed the first one you initially added.
> 
> Tell me whether you're OK with the proposed changes 🙂

Hi @HorlogeSkynet 

Your are welcome 😀. 

Your changes are ok for me. Sorry for the tabs which broke linting, I'll check them better before the next PR.
